src/StatsdClient/Statsd.cs

```diff
@@ -274,6 +275,12 @@ public void Send(string title, string text, string alertType = null, string aggr
             Send(Event.GetCommand(title, text, alertType, aggregationKey, sourceType, dateHappened, priority, hostname, _constantTags, tags, truncateIfTooLong));
         }
 
+        public Task SendAsync(string title, string text, string alertType = null, string aggregationKey = null, string sourceType = null, int? dateHappened = null, string priority = null, string hostname = null, string[] tags = null, bool truncateIfTooLong = false)
+        {
+            truncateIfTooLong = truncateIfTooLong || TruncateIfTooLong;
+            return SendAsync(Event.GetCommand(title, text, alertType, aggregationKey, sourceType, dateHappened, priority, hostname, tags, truncateIfTooLong));

```

Comment:
good catch,  thanks

Reply:
`_constantTags` is missing for every Async function in this file.

Reply:
hey @alistair! Any way I can help out with this last change? Hoping to utilize this feature pretty soon, so I'm happy to help 😄

Reply:
Hey! I think [this PR](https://github.com/alistair/dogstatsd-csharp-client/pull/1) to your fork takes care of the remaining `_constantTags`? Let me know if that looks good, and thanks again for doing this!

Reply:
hey @ogaca-dd, I think this is addressed now?

Reply:
@gdryke: Yes it is!

My plan for next week:
- Merge this PR
- Fix the multithreading issue (https://github.com/DataDog/dogstatsd-csharp-client/pull/59#discussion_r341151335)
- Fix conflict and merge this PR https://github.com/DataDog/dogstatsd-csharp-client/pull/92 (I did not merge it before to avoid creating conflicts in this PR)
- Test everything and create a new package version

I will do my best to make everything next week!